armasec/pluggable/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""
+Manage plugins from armasec.
+"""
+
+import pluggy
+from armasec.pluggable import hookspecs
+
+hookimpl = pluggy.HookimplMarker("armasec")
```

Comment:
I wonder if we should use a more specific name here like `armasec.<something>`, just to leave the door open for the creation of any extra type of plugin in the project.

Reply:
I thought about this, and it's a valid concern. However, for the sake of simplicity (which Armasec aims to provide), I think we don't want to overengineer right now.